hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.tsx

```diff
@@ -38,8 +38,12 @@ interface QuantitySelectorProps {
     handleChange: (e: SelectChangeEvent) => void;
     numInCart: number;
 }
-
-const QuantitySelector = ({
+interface CartCardProps {
```

Comment:
It would be best to keep this defined where it was, right on top of the component that uses it. Type definitions in can go anywhere, and can be referenced by things defined above them just fine if needed